packages/devtools-reps/src/reps/tests/element-node.js

```diff
@@ -320,3 +320,28 @@ describe("ElementNode - Disconnected node", () => {
     expect(renderedComponent.find(".open-inspector").exists()).toBeFalsy();
   });
 });
+
+describe("ElementNode - Element with longString attribute", () => {
+  const stub = stubs.get("NodeWithLongStringAttribute");
+
+  it("selects ElementNode Rep", () => {
+    expect(getRep(stub)).toBe(ElementNode.rep);
+  });
+
+  it("renders with expected text content", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub
+    }));
+
+    expect(renderedComponent.text()).toEqual(`<div data-test="${"a".repeat(1000)}…">`);
```

Comment:
My ocd tells me that we should import and use `ELLIPSIS` instead of `…` in the end of `data-test` attr.

I like the idea of having a small expand button, stylistically similar to attached element events, labeled `show full` or a clickable ellipsis as you mentioned in #835. I'll probably need to update this test once I implement it. :)

Reply:
good catch :)